<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SequenceSelector</class>
 <widget class="QWidget" name="SequenceSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sequence Selector</string>
  </property>
  <widget class="QPushButton" name="loadSequenceButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>400</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Sequence</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>251</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Please select the sequence to view</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>497</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="sequenceRadio_01">
        <property name="text">
         <string>01 - Walking</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sequenceRadio_02">
        <property name="text">
         <string>02 - Crossing</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sequenceRadio_09">
        <property name="text">
         <string>09 - Flying</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sequenceRadio_08">
        <property name="text">
         <string>08 - Grasping</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sequenceRadio_06">
        <property name="text">
         <string>06 - Tips Blending</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sequenceRadio_03">
        <property name="text">
         <string>03 - Crossing and Twisting</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sequenceRadio_05">
        <property name="text">
         <string>05 - Dancing</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sequenceRadio_04">
        <property name="text">
         <string>04 - Tips Touching</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sequenceRadio_07">
        <property name="text">
         <string>07 - Hugging</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sequenceRadio_10">
        <property name="text">
         <string>10 - Rock Gesture</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sequenceRadio_11">
        <property name="text">
         <string>11 - Bunny Gesture</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="sequenceRadio_12">
        <property name="text">
         <string>12 - Bunny Gesture (*)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sequenceRadio_18">
        <property name="text">
         <string>18 - Bending a Rope</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sequenceRadio_17">
        <property name="text">
         <string>17 - Bending a Pipe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sequenceRadio_20">
        <property name="text">
         <string>20 - Moving a Cube with one hand</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sequenceRadio_16">
        <property name="text">
         <string>16 - Moving a Ball with two hands</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sequenceRadio_14">
        <property name="text">
         <string>14 - Rock Gesture (*) </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sequenceRadio_19">
        <property name="text">
         <string>19 - Moving a Ball with one hand and  
occlusion of a manipulating finger</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sequenceRadio_15">
        <property name="text">
         <string>15 - Moving a Ball with one hand</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sequenceRadio_21">
        <property name="text">
         <string>21 - Moving a Cube with one hand and
occlusion of a manipulating finger</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sequenceRadio_13">
        <property name="text">
         <string>13 - Flying (*)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
